pkg/sql/sem/tree/eval.go

```diff
@@ -1651,6 +1651,8 @@ var CmpOps = cmpOpFixups(map[treecmp.ComparisonOperatorSymbol]cmpOpOverload{
 		makeIsFn(types.TimestampTZ, types.Timestamp, volatility.Stable),
 		makeIsFn(types.Time, types.TimeTZ, volatility.Stable),
 		makeIsFn(types.TimeTZ, types.Time, volatility.Stable),
+		makeIsFn(types.Unknown, types.Void, volatility.LeakProof),
```

Comment:
can we fix type resolution to instead evaluate NULL as a VOID? i wonder if this also gets fixed by adding void here (it might not)

https://github.com/cockroachdb/cockroach/blob/c48196c300087cc1682689e7cb6f4231d5bae69a/pkg/sql/sem/tree/constant.go#L480